build-scripts/build_compiler.js

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2018 The Closure Compiler Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Build the compiler java jar and GWT platforms from source.
+ *
+ * Invoked as part of the package build process:
+ *
+ *     yarn run build
```

Comment:
Could you add to this information about the expected state of the `./compiler` submodule when running `yarn run build`?